scripts/install-helm.sh

```diff
@@ -4,12 +4,15 @@
 
 TEMP="/tmp/cephcsi-helm-test"
 
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+# shellcheck source=build.env
+[ ! -e "${SCRIPT_DIR}"/build.env ] || source "${SCRIPT_DIR}"/build.env
```

Comment:
this is wrong, the build.env file is in `SCRIPT_DIR/../`

Reply:
removed ready-to-merge label

Reply:
This is correct isnt it. the build.env is in main directory

Reply:
sorry i missed this one, now its correct as build.env is outside of script dir